### DISCUSSION TITLE
FOX HD | (Slovenia)

### DISCUSSION THREAD

**Discussion by R4IN2007** [iptv-org/community#59] [Show and tell]
### Channel Name

FOX HD

### Channel ID

FoxSerbia.rs

### Stream URL

http://194.163.179.246/slovenci/filmski/fox.m3u8

### Please confirm the following

- [X] I have read [Contributing Guide](https://github.com/iptv-org/iptv/blob/master/CONTRIBUTING.md#request-a-channel)

**Comment by freearhey** [comment]
<img width="669" alt="image" src="https://user-images.githubusercontent.com/7253922/185268443-447a226d-6f78-4ec1-8864-88a725b39258.png">
